### PR TITLE
Yamlfmt hook

### PR DESCRIPTION
**Pull request by jumanjiman**

### CODE REVIEW

File: all-repos.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@
 - https://github.com/elidupuis/mirrors-jscs
 - https://github.com/elidupuis/mirrors-sass-lint
 - https://github.com/jumanjihouse/pre-commit-hooks
+- https://github.com/jumanjihouse/yamlfmt
```
Comment (asottile) [inline all-repos.yaml]: huh this is confusing, there's another tool called `yamlfmt` in python (and it's different from this one?): https://github.com/mmlb/yamlfmt

File: all-repos.yaml [hunk 2]
```diff
@@ -64,6 +65,7 @@
 - https://github.com/psf/black
 - https://github.com/IamTheFij/ansible-pre-commit
 - https://github.com/IamTheFij/docker-pre-commit
+- https://github.com/macisamuele/language-formatters-pre-commit-hooks
```
Comment (asottile) [inline all-repos.yaml]: these intentionally aren't alphabetized -- they're ordered by time-of-addition and then grouped by organization (so this should be added at the end)

File: all-repos.yaml (hunk 1 above)
Comment (asottile) [reply all-repos.yaml]: additionally, the package being added here clobbers the `pre_commit_hooks` python namespace (and should pick a different name/layout)

Comment (jumanjiman) [reply all-repos.yaml]: should i rename my repo as `pre-commit-hook-yamlfmt` and update this?

do you mean that https://github.com/jumanjihouse/yamlfmt/blob/master/setup.py#L12 clobbers the namespace?
I can fix that as well!

File: all-repos.yaml (hunk 2 above)
Comment (jumanjiman) [reply all-repos.yaml]: GTK - i'll update

Comment (jumanjiman) [reply all-repos.yaml]: pushed this fix to put at the end

File: all-repos.yaml (hunk 1 above)
Comment (asottile) [reply all-repos.yaml]: yep, both of those sound good

Comment (jumanjiman) [reply all-repos.yaml]: updated

Comment (jumanjiman) [reply all-repos.yaml]: from https://github.com/jumanjihouse/pre-commit-hook-yamlfmt/blob/master/setup.py#L13

```
    # globally-unique name
    packages=[
        'jumanjihouse_precommit_yamlfmt',
    ],
```

**Comment by jumanjiman** [comment]
tyvm!

**Pull request merged**